back/src/domain/offer/useCases/SendEmailsWithAssessmentCreationLink.ts

```diff
@@ -101,6 +101,7 @@ export class SendEmailsWithAssessmentCreationLink extends TransactionalUseCase<
       params: {
         agencyLogoUrl: agency.logoUrl,
         agencyValidatorEmail: agency.validatorEmails[0],
+        agencyImmersionAppraisalLink: agency.questionnaireUrl,
```

Comment:
```suggestion
        agencyAssessmentDocumentLink: agency.questionnaireUrl,
```

Le terme qu'on utilise généralement est assessment.